### PR TITLE
feat(cli): bootstrap shows different prompt for bootstrapped clusters

### LINKED ISSUE #746: `glasskube bootstrap` should show different prompt when updating
**Is your feature request related to a problem? Please describe.**
Currently the `glasskube bootstrap` command shows a prompt like "Glasskube will be installed in context xyz. Continue?" (see #719). While this is correct for non-bootstrapped clusters, it makes no sense when the user wants to update the current glasskube installation and might be confusing. 

**Describe the solution you'd like**
When glasskube is bootstrapped in the current cluster, the prompt should be: "Glasskube will be updated to version [version] in cluster [cluster]. Continue?" 

As in #719, if the `--yes` flag is set, the prompt should not appear. 

There is already a check in place, whether the installation would be downgraded. This check should be done first. If the installation would not be downgraded, then the prompt for installing/updating should appear, because only at this point we know if glasskube is already installed and in which version. 

**Describe alternatives you've considered**

**Additional context**

### PR DESCRIPTION
**Pull request by utsab818**
Closes #746 

## 📑 Description
`glasskube bootstrap` command for bootstrapped-clusters now give updated prompt to avoid confusion.
Most cases when upgrading the version is handled with proper prompt.

## ✅ Checks
- [ ] My code requires changes to the documentation
- [ ] I have updated the documentation as required

### CODE REVIEW

**Review by christophenne — COMMENTED** [review]
Thanks for your PR!

Looks good, I noticed two things when testing, see comments. Please let me know if you want to incorporate these yourself or if we should open up another issue.

**Comment by utsab818** [comment]
@christophenne I will definitely try to incorporate those lines of codes.

**Comment by utsab818** [comment]
@christophenne Does this look good or is there any other cases or any other proper methods to do so?
image

**Comment by christophenne** [comment]
@utsab818 the check for installing the same version doesn't work, because `semver.IsUpgradable(installedVersion, desiredVersion)` returns false when the two versions are the same. So this check should be outside of the `upgradeNeeded` if, I think. The rest works fine. 

Testing hint: 
* `minikube start -p bootstrap-test`
* change `var Version` in `internal/config.go` to `"0.8.0"` (for example)
* `go run cmd/glasskube/main.go bootstrap`

Then you can change the version as you wish and test all the cases. When done, delete the minikube cluster with `minikube delete -p bootstrap-test`.

**Comment by utsab818** [comment]
@christophenne Thanks for the hint. I will update it.

**Comment by christophenne** [comment]
> @christophenne Thanks for the hint. I will update it.

Thanks! However, now bootstrap doesn't work in an empty cluster, I think you should additionally check for `installedVersion != ""` in that `if`: 

```
panic: runtime error: slice bounds out of range [1:0]

goroutine 1 [running]:
github.com/glasskube/glasskube/cmd/glasskube/cmd.init.func1(0x3221140, {0x32a2300?, 0x4?, 0x1e63fed?})
        /home/christoph/dev/glasskube/cmd/glasskube/cmd/bootstrap.go:99 +0x92e
github.com/spf13/cobra.(*Command).execute(0x3221140, {0x32a2300, 0x0, 0x0})
```

**Comment by utsab818** [comment]
@christophenne Sorry for causing you trouble with these small things. 
I have added installedVersion != "" in elseif.
Is there any other criteria that needs to be looked for?
I tried doing some tests and seems to be working fine.

File: cmd/glasskube/cmd/bootstrap.go [hunk 1]
```diff
@@ -82,6 +75,43 @@ var bootstrapCmd = &cobra.Command{
...
+					}
+				} else {
+					confirmMessage := fmt.Sprintf("Glasskube will be updated to version %s "+
+						"in cluster %s.\nContinue? ", desiredVersion, currentContext)
+					if !cliutils.YesNoPrompt(confirmMessage, true) {
+						fmt.Println("Operation stopped")
+						cliutils.ExitWithError()
+					}
+				}
+			} else if installedVersion[1:] == desiredVersion {
```
Comment (christophenne) [inline cmd/glasskube/cmd/bootstrap.go:99]: `installedVersion` could be empty here, and then the `[1:]` access panics. So this condition should be `installedVersion != "" && ...`

**Comment by christophenne** [comment]
> @christophenne Sorry for causing you trouble with these small things. I have added installedVersion != "" in elseif. Is there any other criteria that needs to be looked for? I tried doing some tests and seems to be working fine.

No problem, did you maybe not commit/push it?

The test case I tried is simply a bootstrap in a new empty cluster. This will lead to `installedVersion` being empty.

**Comment by utsab818** [comment]
@christophenne I will push it now. I was just waiting for your confirmation.

**Review by christophenne — APPROVED** [review]

**Pull request merged**